src/applications/disability-benefits/all-claims/content/contactInformation.jsx

```diff
@@ -63,6 +58,20 @@ const EmailViewField = ({ formData, name }) => (
   </p>
 );
 
+const EffectiveDateViewField = ({ formData }) => {
+  const { from, to } = formData;
+  const dateFormat = 'MMM D, YYYY';
+  const fromDateString = moment(from).format(dateFormat);
+  return to ? (
+    <p>
+      We’ll use this address starting on {`${fromDateString} `}
```

Comment:
I feel like this is another case where Prettier makes not-so-pretty formatting decisions. :man_shrugging:

Reply:
Yeah...I figured out how to make this better.